tests/utils/object_store/test_gs_object_store.py

```diff
@@ -7,8 +7,40 @@
 
 import pytest
 from botocore.exceptions import ClientError
+from torch.utils.data import DataLoader
 
+from composer.trainer import Trainer
 from composer.utils import GCSObjectStore
+from tests.common import RandomClassificationDataset, SimpleModel
+
+
+@pytest.mark.remote
+def test_gs_object_store_integration_json_auth():
```

Comment:
can you assert in here that the object store that gets created uses the sdk version?

Reply:
since you use this for both tests, probs need a flag for which version its supposed to use, but basically just check that you're actually testing both versions.

Reply:
ok added a check